packages/theme-light-extension/style/variables.css

```diff
@@ -419,6 +419,6 @@ all of MD as it is not optimized for dense, information rich UIs.
   --jp-text-editor-icon-color: var(--md-grey-700);
   --jp-inspector-icon-color: var(--md-grey-700);
   --jp-switch-color: var(--md-grey-400);
-  --jp-switch-true-position-color: var(--md-orange-700);
+  --jp-switch-true-position-color: var(--md-orange-900);
   --jp-switch-cursor-color: rgba(255, 255, 255, 1);
```

Comment:
I did not find any use of `--jp-switch-cursor-color`. If it is indeed unused could your remove it from the two themes please.

Reply:
Thanks. You are right. This css variable has been removed in commit 
cd90e73.